lib/livebook/runtime/erl_dist/runtime_server.ex

```diff
@@ -644,4 +644,29 @@ defmodule Livebook.Runtime.ErlDist.RuntimeServer do
       end)
     end)
   end
+
+  defp normalize_path_env(env_vars) when is_list(env_vars) do
+    env_vars |> Enum.into(%{}) |> normalize_path_env()
+  end
+
+  defp normalize_path_env(env_vars) when is_map(env_vars) do
```

Comment:
According to the runtime protocol it should always be a list :)

Reply:
Good call